src/SFA.DAS.AssessorService.Application.Api.Client/Clients/ApiClientBase.cs

```diff
@@ -84,7 +84,7 @@ private static HttpRequestException CreateRequestException(HttpRequestMessage fa
                     failedResponse.Content.ReadAsStringAsync().Result));
         }
 
-        protected async Task<T> RequestAndDeserialiseAsync<T>(HttpRequestMessage request, string message = null) where T : class
```

Comment:
Structs are valid JSON too! We have instances where an API call returns a struct  (i.e. true).

I've removed the constraint as otherwise I'd be creating an identical version of this method for structs.